addon/templates/components/uni-tabs.hbs

```diff
@@ -0,0 +1,7 @@
+<div class="uni-tabs__wrapper">
+    {{#each tabs as |item|}}
+        <div class="uni-tabs__wrapper__item">
+            <a href="{{item.url}}">{{item.label}}</a>
```

Comment:
this should be a link-to. this way, when a specific route is active, it automatically triggers the active class.
https://guides.emberjs.com/v2.8.0/templates/links/